server/src/constants/botId.js

```diff
@@ -0,0 +1,3 @@
+const botId = '5d326949e38a33679e223b07';
```

Comment:
Where are we using this botId?
Could we identify by username or some email id if it is really required.
Because this Id will change after each new mongodb setup.

Reply:
UserId and Email may change as well
Kept Id as it is already indexed by mongodb

Reply:
No.... I mean if you got new db, you can add user with same email id yourself. but Id is auto created by mongodb... So it will always change.

Reply:
It will be a part of the configuration right?
added it to env